assets/section-image-banner.css

```diff
@@ -296,7 +301,6 @@
 
 @media screen and (min-width: 750px) {
   .banner__box {
-    padding: 5rem;
     width: auto;
     max-width: 71rem;
     min-width: 45rem;
```

Comment:
For visibility Id add it to the description of the PR which sections are affected which I believe is only the banner and multi column

Reply:
and email sign up banner (password page)